test/tick-processor/test-tick-processor-polyfill-brokenfile.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (!common.enoughTestCpu)
+  common.skip('test is CPU-intensive');
+
+if (common.isWindows ||
+  common.isSunOS ||
+  common.isAIX ||
+  common.isLinuxPPCBE ||
+  common.isFreeBSD)
+  common.skip('C++ symbols are not mapped for this os.');
+
+//This test will produce a broken profile log.
+//ensure prof-polyfill not stuck in infinite loop
+//and success process
+
+
+const assert = require('assert');
+const cp = require('child_process');
+const path = require('path');
+const fs = require('fs');
+
+const LOG_FILE = path.join(tmpdir.path, 'tick-processor.log');
+const RETRY_TIMEOUT = 150;
+const BORKEN_PART = 'tick,0x100840650';
```

Comment:
typo `s/BORKEN_PART/BROKEN_PART`

Also, could we just use `const BROKEN_PART = 'tick,'/` instead? `"0x100840650"` looks like a magic number here.

Reply:
yes,it's better.